react/components/UserDropdown/components/MyGroups/fragments/myGroups.js

```diff
@@ -6,7 +6,7 @@ export default gql`
   fragment MyGroups on Me {
     __typename
     id
-    is_my_groups_dropdown_visible: flag(name: "is_my_groups_dropdown_visible")
+    is_my_groups_dropdown_hidden: flag(name: "is_my_groups_dropdown_hidden")
```

Comment:
I know this is slightly weird, but we have falsey nils by default and this needs to be open by default.

Reply:
👍